Formula/git.rb

```diff
@@ -5,6 +5,7 @@ class Git < Formula
   url "https://www.kernel.org/pub/software/scm/git/git-2.27.0.tar.xz"
   sha256 "73ca9774d7fa226e1d87c1909401623f96dca6a044e583b9a762e84d7d1a73f9"
   head "https://github.com/git/git.git", :shallow => false
+  revision 1
```

Comment:
I don’t think this needs a revision bump.

Reply:
I think I agree. I was told, previously, that there should be a revision bump for style improvements (such as `Utils.safe_popen_read` instead of `Utils.popen_read`) which is why I included this here.

Reply:
Revision bump should really only be for things that would warrant forcing the user to update their install.